draft-irtf-cfrg-hpke.md

```diff
@@ -375,133 +375,126 @@ and {{encryption-context}} for details.
 
 ## DH-Based KEM {#dhkem}
 
-Suppose we are given a KDF, and the following interface to a cyclic group `G` (and hash 
-function `Hash`). Implementations of this interface based on the P-256, P-384 and the
-P-521 family of NIST curves as well as the Edwards25519 and Edwards448 curves can be 
-found in {{ciphersuites}}.
+Suppose we are given a KDF, and the following interface to a cyclic group `G`. Implementations of this interface based on the P-256, P-384 and the
+P-521 family of NIST curves as well as the Edwards25519 and Edwards448 curves are described in in {{ciphersuites}}.
 
 * Data Types:
-  - element of field `F` : finite field of "scalars".
-  - element of group `G` : additive group of "points" over `F`.
+  - element of field `F` : finite field of “scalars”.
+  - element of group `G` : additive group of “points” over `F`.
 
 * Constants:
   - `B` : generator of group `G`
-  - `2n` : length in octets (rounded to nearest even integer) of encoding of a field
-  element in `F`
-  - `ptLen` : length, in octets, of a point encoded as an octet string
-  - `hLen` : length, in octets, of hash function's output
+  - `Nf` : length in bytes (rounded to nearest even integer) of a field
+    element in `F` encoded as a byte string
+  - `Np` : length in bytes of a point encoded as a byte string
```

Comment:
I suggest changing to `Np` because people might confuse `pt` with plaintext, as this is the variable name used for plaintexts in HPKE, see https://github.com/cfrg/draft-irtf-cfrg-hpke/blob/master/draft-irtf-cfrg-hpke.md#single-shot-apis.